hassio/hassio-navigation.html

```diff
@@ -0,0 +1,50 @@
+<link rel='import' href='../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../bower_components/paper-tabs/paper-tabs.html'>
+<link rel='import' href='../bower_components/paper-tabs/paper-tab.html'>
+
+<link rel='import' href='../src/util/hass-mixins.html'>
+
+<dom-module id='hassio-navigation'>
+  <template>
+    <style include='iron-flex iron-positioning ha-style'>
+      paper-tabs {
+        margin-left: 12px;
+        --paper-tabs-selection-bar-color: #FFF;
+        text-transform: uppercase;
+      }
+    </style>
+    <paper-tabs
+      scrollable
+      selected='[[page]]'
+      attr-for-selected='page-name'
+      on-iron-activate='handlePageSelected'
+    >
+      <paper-tab page-name='dashboard'>Dashboard</paper-tab>
+      <paper-tab page-name='store'>Add-on store</paper-tab>
+      <paper-tab page-name='snapshots'>Snapshots</paper-tab>
+      <paper-tab page-name='advanced'>Advanced</paper-tab>
+    </paper-tabs>      
+  </template>
+</dom-module>
+
+<script>
+class HassioNavigation extends window.hassMixins.EventsMixin(Polymer.Element) {
+  static get is() { return 'hassio-navigation'; }
+
+  static get properties() {
+    return {
+      page: String,
+    };
+  }
+
+  handlePageSelected(ev) {
+    const newPage = ev.detail.item.getAttribute('page-name');
+    if (newPage && newPage !== this.page) {
+      history.pushState(null, null, '/hassio/' + newPage);
```

Comment:
These 2 lines can be replaced by `this.naviate(`/hassio/${newPage}`) if you use the `NavigateMixin` instead of the `EventsMixin`